e2e/machine_migration_test.go

```diff
@@ -111,6 +112,72 @@ var _ = Describe("[sig-cluster-lifecycle][OCPFeatureGate:MachineAPIMigration] Ma
 			})
 		})
 	})
+
+	var _ = Describe("Deleting MAPI/CAPI Machines", Ordered, func() {
+		var mapiMachineAuthCAPIName = "machine-authoritativeapi-capi-deletion"
+		var newCapiMachine *clusterv1.Machine
+		var newMapiMachine *machinev1beta1.Machine
+		var err error
+
+		Context("with spec.authoritativeAPI: ClusterAPI and delete MAPI machine", func() {
```

Comment:
Are you planning to also add cases for authoritativeAPI: MachineAPI like so?
- MAPI Authoritative - Delete MAPI Machine
- MAPI Authoritative - Delete CAPI Machine

Reply:
Yes, we will. I think @miyadav is working on add cases for authoritativeAPI: MachineAPI.